core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -69,10 +80,50 @@ static PageStore create(AlluxioConfiguration conf) {
         throw new IllegalArgumentException(String.format("Unrecognized store type %s",
             storeType.name()));
     }
-    options.setRootDir(conf.get(PropertyKey.USER_CLIENT_CACHE_DIR));
+    String rootDir = conf.get(PropertyKey.USER_CLIENT_CACHE_DIR);
+    initialize(rootDir, storeType);
+    Path storePath = getStorePath(storeType, rootDir);
+    options.setRootDir(storePath.toString());
+    options.setPageSize(conf.getBytes(PropertyKey.USER_CLIENT_CACHE_PAGE_SIZE));
+    options.setCacheSize(conf.getBytes(PropertyKey.USER_CLIENT_CACHE_SIZE));
+    options.setAlluxioVersion(conf.get(PropertyKey.VERSION));
     return create(options);
   }
 
+  /**
+   * Gets store path given root directory and store type.
+   *
+   * @param storeType the type of the page store
+   * @param rootDir the root directory path
+   * @return the store directory path
+   */
+  static Path getStorePath(PageStoreType storeType, String rootDir) {
+    return Paths.get(rootDir, storeType.name());
+  }
+
+  /**
+   * Initialize a page store at the configured location.
+   * Data from different store type will be removed.
+   *
+   * @param rootPath root path of the page store
+   * @param storeType the page store type
+   */
+  static void initialize(String rootPath, PageStoreType storeType) throws IOException {
+    Path storePath = getStorePath(storeType, rootPath);
+    Files.createDirectories(storePath);
+    LOG.info("Clean cache directory {}", rootPath);
+    try (Stream<Path> stream = Files.list(Paths.get(rootPath))) {
+      stream.filter(path -> !storePath.equals(path))
+          .forEach(path -> {
+            try {
+              FileUtils.deletePathRecursively(path.toString());
+            } catch (IOException e) {
+              LOG.warn("failed to delete {} in cache directory: {}", path, e.getMessage());
```

Comment:
same case for https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#warn-log-level

Reply:
Done.